draft-ounsworth-pq-composite-sigs.mkd

```diff
@@ -93,11 +94,10 @@ The deployment of composite public keys and composite signatures using post-quan
 - Algorithm strength uncertainty: During the transition period, some post-quantum signature and encryption algorithms will not be fully trusted, while also the trust in legacy public key algorithms will start to erode.  A relying party may learn some time after deployment that a public key algorithm has become untrustworthy, but in the interim, they may not know which algorithm an adversary has compromised.
 - Backwards compatibility: During the transition period, post-quantum algorithms will not be supported by all clients.
 
-This document provides a mechanism to address algorithm strength uncertainty by providing formats for encoding multiple public keys and multiple signature values into existing public key and signature fields, as well as an algorithm for validating a composite signature. The issue of backwards compatibility is left open to be addressed in separate draft(s).
+This document provides a mechanism to address algorithm strength uncertainty by providing formats for encoding multiple public keys and multiple signature values into existing public key and signature fields, as well as an algorithm for validatinfg a composite signature. The issue of backwards compatibility is left open to be addressed in separate draft(s).
```

Comment:
typo introduced - validatinfg

Reply:
nit: "public keys and signatures" - it also describes how to encode private key